di_website/templates/publications/footnotes.html

```diff
@@ -7,7 +7,11 @@
                 <h2 class="heading type-l type-l--trailer">Notes</h2>
                 {% for item in page.footnotes_list %}
                 <ul class="footnotes__list">
+                {% if forloop.counter <= 99 %}
                     <li class="footnotes__item">
+                {% else %}
+                    <li class="footnotes__item three-digit-footnote">
+                {% endif %}
```

Comment:
Inline this `<li class="footnotes__item {% if forloop.counter > 99 %}three-digit-footnote{% endif %}">`